src/pyTENAX/pyTENAX.py

```diff
@@ -172,7 +174,7 @@ def remove_incomplete_years(self, data_pr, name_col = 'value', nan_to_zero=True)
         # Step 1: get resolution of dataset (MUST BE SAME in whole dataset!!!)
         time_res = (data_pr.index[-1] - data_pr.index[-2]).total_seconds()/60
         # Step 2: Resample by year and count total and NaN values
-        yearly_valid = data_pr.resample('Y').apply(lambda x: x.notna().sum())  # Count not NaNs per year
+        yearly_valid = data_pr.resample('YE').apply(lambda x: x.notna().sum())  # Count not NaNs per year
```

Comment:
This change from "Y" to "YE" (year-end) will cause an error "*** ValueError: Invalid frequency: YE" in older versions of pandas.
Does "Y" not work in new version of pandas? 
We must start to be somehow consistent in our conda environments to prevent these issues cause we later need to provide requirements.txt

Reply:
check pd.__version__ , mine is '2.1.3' and I think they changed Y to YE in 2.2+